secure/storage/src/value.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(test)]
+use crate::error::Error;
+use libra_crypto::ed25519::Ed25519PrivateKey;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, PartialEq, Serialize)]
+#[serde(content = "value", rename_all = "snake_case", tag = "type")]
+pub enum Value {
```

Comment:
is this test only or generate struct for any value that might be stored? how to extend the Value struct if it's the latter case?

Reply:
Append only and no problems.